bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -1507,7 +1528,7 @@ void BcStTest::getReservedPagesStage(bool skipReply, bool reject, size_t sleepDu
     const auto& msg = testedReplicaIf_.sent_messages_.front();
     ASSERT_NFF(assertMsgType(msg, MsgType::FetchResPages));
     auto fetchResPagesMsg = reinterpret_cast<FetchResPagesMsg*>(msg.data_.get());
-    RejectFetchingMsg outMsg;
+    RejectFetchingMsg outMsg(RejectFetchingMsg::Reason::RES_PAGE_NOT_FOUND);
```

Comment:
nit: to support future changes, should we add a reject code to getReservedPagesStage? wuth current default RES_PAGE_NOT_FOUND?